pool/hub.go

```diff
@@ -339,6 +339,14 @@ func (h *Hub) Listen() error {
 	return nil
 }
 
+// CloseListeners terminates listeners created by endpoints of the hub. This
+// should only be used in the pool's shutdown process the hub is not running.
+func (h *Hub) CloseListeners() {
```

Comment:
This is being called if an error occurs during creating the GUI - nice. Does it also need to be called when the application is shutting down after starting successfully (eg. CTRL+C)?

Reply:
Nope, context cancellation will take care of closing the listeners and terminating all running processes if the pool starts successfully.